plugins/modules/ec2_vol.py

```diff
@@ -488,6 +494,9 @@ def create_volume(module, ec2_conn, zone):
             if multi_attach:
                 additional_params['MultiAttachEnabled'] = True
 
+            if outpost_arn:
```

Comment:
We should add an extra check here to see if the supplied outpost_arn matches the regex that AWS defines, and return as error if it does not.
The actual regex function should go in a module_util so that we can reuse it in the future, and a unit test be added to verify the regex.